### PR TITLE
adding windows to travis - fixing goveralls

### PR DESCRIPTION
**Pull request by aricart**
- Added windows build on travis
- Fixed missing goveralls

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -29,8 +29,13 @@ after_success:
 
 deploy:
 - provider: script
-  skip_cleanup: true
+  cleanup: false
```
Comment (kozlovic) [inline .travis.yml:32]: May work, but I usually set `cleanup: true` which is an equivalent of `git stash`, basically cleaning the env. If you don't do that, the go releaser may fail to publish the release saying that git state is dirty.

File: .travis.yml [hunk 2]
```diff
@@ -29,8 +29,13 @@ after_success:
...
   script: curl -sL http://git.io/goreleaser | bash
   on:
     tags: true
     condition: $TRAVIS_OS_NAME = linux
+- provider: script
+  script: $HOME/gopath/bin/goveralls -coverprofile=v2/coverage.out -service travis-ci
+  on:
+    tags: true
+    condition: $TRAVIS_OS_NAME = linux && $TRAVIS_GO_VERSION =~ ^1.12.
```
Comment (kozlovic) [inline .travis.yml:41]: Newline maybe? Also, should you update the top and here to a more recent version of Go?

File: .travis.yml (hunk 1 above)
Comment (aricart) [reply .travis.yml]: fixed

File: .travis.yml (hunk 2 above)
Comment (aricart) [reply .travis.yml]: fixed

File: .travis.yml [hunk 3]
```diff
@@ -16,22 +16,22 @@ install:
...
 script:
     go test -coverpkg=./... -coverprofile=./coverage.out ./...
 
-after_success:
-- git reset --hard
-
 deploy:
 - provider: script
-  skip_cleanup: true
+  script: $HOME/gopath/bin/goveralls -coverprofile=v2/coverage.out -service travis-ci
```
Comment (kozlovic) [inline .travis.yml:28]: why `v2/coverage.out`?

File: .travis.yml [hunk 4]
```diff
@@ -16,22 +16,22 @@ install:
...
+  on:
+    tags: true
```
Comment (kozlovic) [inline .travis.yml:30]: So you would publish coverage only for tag pushes?

File: .travis.yml (hunk 3 above)
Comment (aricart) [reply .travis.yml]: c/p err

File: .travis.yml (hunk 4 above)
Comment (aricart) [reply .travis.yml]: changed to always

**Pull request merged**